docs/Reference/Pantheon-CLI-Syntax.md

```diff
@@ -258,8 +280,41 @@ metrics. The default is `127.0.0.1`. The metrics server respects the [`--host-wh
 metrics-port="6174"
 ```
 
-Specifies the port on which the [Prometheus](https://prometheus.io/) monitoring service accesses Pantheon
-metrics. The default is `9545`. The metrics server respects the [`--host-whitelist` option](#host-whitelist).
+Specifies the port on which [Prometheus](https://prometheus.io/) accesses [Pantheon metrics](../Using-Pantheon/Debugging.md#monitor-node-performance-using-prometheus) 
+or which Pantheon pushes metrics to a [Prometheus Pushgateway](https://github.com/prometheus/pushgateway).
+The default is `9545`. 
```

Comment:
Again, with #678 this will have its own `--metrics-push-port` arg.

Reply:
Done